elasticsearch_rebalancer/__init__.py

```diff
@@ -198,7 +198,7 @@ def print_execute_reroutes(es_host, commands):
         raise BalanceException('User exited serial rerouting!')
 
     cluster_update_interval = get_transient_cluster_settings(
-        es_host, 'cluster.info.update.interval',
+        es_host, ['cluster.info.update.interval'],
```

Comment:
does this being a list or a string depend on the version of ES we're using?

Reply:
no: https://github.com/EDITD/elasticsearch-rebalancer/blob/develop/elasticsearch_rebalancer/util.py#L63

Reply:
ok cool